modules/documentation/src/site/xdoc/userguide/samples/sample705.xml

```diff
@@ -58,6 +58,7 @@
       &lt;parameter name="max.deliver.attempts"&gt;3&lt;/parameter&gt;
       &lt;parameter name="max.deliver.drop"&gt;true&lt;/parameter&gt;
       &lt;parameter name="retry.http.status.codes"&gt;500, 504&lt;/parameter&gt;
+      &lt;parameter name="retry.interval"&gt;5000&lt;/parameter&gt;
```

Comment:
It would be good if you can adopt the default value that is in the code [1] 

[1]https://github.com/apache/synapse/pull/41/files#diff-af12a51dc14c81dc0feaf0cfa7e8667aR88

Reply:
I have updated the PR. Thank for the feedback.